editor/gui/scene_tree_editor.cpp

```diff
@@ -599,45 +714,156 @@ void SceneTreeEditor::_set_item_custom_color(TreeItem *p_item, Color p_color) {
 }
 
 void SceneTreeEditor::_node_script_changed(Node *p_node) {
```

Comment:
Looks like another signal that could be deprecated 🤔

Reply:
Yeah, I could do a follow-up PR to make some changes to the signals we expose. Ideally I'd add some parameters to the editor_state_changed signal to streamline it down further. I was trying to not break existing users of these signals in this PR so it could be backported. I could do a "breaking" PR for 4.4 or 4.5 to consolidate the signals?

Reply:
Sounds good.